lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -200,8 +200,12 @@ def main():
         if increase_psf:
             set_numba_seed(input.root.dl1.event.subarray.trigger.col('obs_id')[0])
 
+        image_mask_save = not args.noimage and 'image_mask' in input.root[dl1_images_lstcam_key].colnames
+
         with tables.open_file(args.output_file, mode='a') as output:
             params = output.root[dl1_params_lstcam_key].read()
+            if image_mask_save:
+                image_mask = output.root[dl1_images_lstcam_key].col('image_mask')
```

Comment:
Do we need the previously applied mask? Just the new one is relevant, right?

Reply:
I think so, since the parameters are changed, the previous mask is not very relevant anymore and can be replaced imo.

Reply:
for the record, this is the implemented behaviour: the new clean masks replaces the old ones